pkg/activator/throttler.go

```diff
@@ -70,8 +61,9 @@ func (t *Throttler) Remove(rev RevisionID) {
 
 // UpdateCapacity update the max concurrency of the Breaker corresponding to the revision
```

Comment:
Should this mention that it creates a new breaker?

Reply:
Yeap